styles/components/_formErrors.scss

```diff
@@ -0,0 +1,21 @@
+.errorList {
```

Comment:
I'm sure this could be improved. I imagine adjusting the markup in the `formError` template to use some the structure classes you created would make a bunch this css unnecessary. Feel free to adjust or point me in the right direction (I haven't spent much time playing with your classes as of yet).